openadr-client/tests/event.rs

```diff
@@ -0,0 +1,204 @@
+use axum::http::StatusCode;
+
+use openadr_client::{Error, PaginationOptions};
+use openadr_wire::{
+    event::{EventContent, Priority},
+    program::ProgramId,
+    target::TargetLabel,
+};
+
+mod common;
+
+fn default_content() -> EventContent {
+    EventContent {
+        object_type: None,
+        program_id: ProgramId::new("program_id").unwrap(),
+        event_name: Some("event_name".to_string()),
+        priority: Priority::MAX,
+        report_descriptors: None,
+        interval_period: None,
+        intervals: vec![],
+        payload_descriptors: None,
+        targets: None,
+    }
+}
+
+#[tokio::test]
+async fn get() {
+    let client = common::setup_client();
+    let event_client = client.create_event(default_content()).await.unwrap();
+
+    assert_eq!(event_client.content(), &default_content());
+}
+
+#[tokio::test]
+#[ignore]
+async fn delete() {
```

Comment:
this fails because `EventName` is used to filter, (as defined in `Target::target_label`). So, should  we implement that, and then delete all events that match the event name, or maybe refine further somehow so that only the event we currently have a handle to gets deleted?